### DISCUSSION TITLE
osu!lazer on pop-os(linux) doesn't feel like it runs as well as it should

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#14678] [Q&A] [answered]
so i moved operating systems from windows to pop-os and i feel like windows feels a lot better when playing than pop-os
(ask for details because i'm not exactly sure what to provide)

here are logs:
[updater.log](https://github.com/ppy/osu/files/7126666/updater.log)
[runtime.log](https://github.com/ppy/osu/files/7126667/runtime.log)
[performance-update.log](https://github.com/ppy/osu/files/7126668/performance-update.log)
[performance-gamethread.update.log](https://github.com/ppy/osu/files/7126669/performance-gamethread.update.log)
[performance-gamethread.input.log](https://github.com/ppy/osu/files/7126670/performance-gamethread.input.log)
[performance-gamethread.draw.log](https://github.com/ppy/osu/files/7126671/performance-gamethread.draw.log)
[performance-gamethread.audio.log](https://github.com/ppy/osu/files/7126672/performance-gamethread.audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/7126673/performance-draw.log)
[performance-audio.log](https://github.com/ppy/osu/files/7126674/performance-audio.log)
[performance.log](https://github.com/ppy/osu/files/7126676/performance.log)
[network.log](https://github.com/ppy/osu/files/7126677/network.log)
[database.log](https://github.com/ppy/osu/files/7126678/database.log)

osu!lazer version: 2021.907.0

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/11800

**Comment by smoogipoo** [comment]
As an interim solution, you can run osu! with the environment variable `COMPlus_GCGen0MaxBudget=600000`. It should bring perf mostly in-line with Windows.
